views/clickhouse.go

```diff
@@ -60,7 +61,17 @@ func NewClickhouseConn(opts ClickhouseOptions) (*ClickhouseClient, error) {
 func (c *ClickhouseClient) QueryRealtimeViewsEvents(ctx context.Context, spec QuerySpec) ([]RealtimeViewershipRow, error) {
 	sql, args, err := buildRealtimeViewsEventsQuery(spec)
 	if err != nil {
-		return nil, fmt.Errorf("error building viewership events query: %w", err)
+		return nil, fmt.Errorf("error building realtime viewership events query: %w", err)
+	}
+	var res []RealtimeViewershipRow
+	err = c.conn.Select(ctx, &res, sql, args...)
+	return res, err
```

Comment:
WDYT of returning only 1 object from this query, instead of an array? Do you see us needing any breakdown on this API at some point? i.e. showing real-time geographic data? (if we do, might as well keep the array not to break stuff later)

Reply:
I think we do need an array, because you may query with the `breakdownBy[]`, then you receive an array of different dimensions.